src/pipecat/services/tts_service.py

```diff
@@ -455,13 +497,24 @@ async def _process_text_frame(self, frame: TextFrame):
         text: Optional[str] = None
         if not self._aggregate_sentences:
             text = frame.text
+            aggregated_by = "token"
```

Comment:
Just to understand, why is the aggregated_by here "token", and not AggregationType.WORD ?

Reply:
i went this way because if the internal aggregation is turned off and we get here, then we are literally just passing on `LLMTextFrame`s which are token-by-token. Which... is typically word? So I was on the fence with what to call this one and whether `"token"` should be added as one of the built-in aggregated types.

Reply:
invitation to comment: @kompfner @aconchillo